src/components/site/SiteHeader.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 
 const SiteHeader = props => {
-  return <p>Placeholder</p>;
+  return <header className="header">{props.children}</header>;
```

Comment:
This class should be `dg_header`. `dg` represents the namespace for this site. This is done to avoid conflicts that may occur. See [here](https://github.com/baltimorecounty/dotgov-components/wiki/Development-Guidelines#sass--css) for more. (yes i realize it still needs to be expanded, will do that).

Reply:
will change on all classes